src/webgl/camera.js

```diff
@@ -141,6 +141,12 @@ p5.prototype.perspective = function(fovy,aspect,near,far) {
  *
  */
 p5.prototype.ortho = function(left,right,bottom,top,near,far) {
+  left = arguments[0] = arguments[0] || (-this.width/2);
```

Comment:
Might create a problem later on if I do this
https://github.com/processing/p5.js/blob/master/src/webgl/camera.js#L146